cli/internal/helm/client.go

```diff
@@ -94,8 +94,11 @@ func (c *Client) currentVersion(release string) (string, error) {
 	return rel[0].Chart.Metadata.Version, nil
 }
 
+// ErrConfirmationMissing signals that an action requires user confirmation.
+var ErrConfirmationMissing = errors.New("action requires user confirmation")
+
 func (c *Client) upgradeRelease(
-	ctx context.Context, timeout time.Duration, chartPath, releaseName string, hasCRDs bool,
+	ctx context.Context, timeout time.Duration, chartPath, releaseName string, hasCRDs bool, confirmation bool,
```

Comment:
Calling the field confirmation might be a bit unspecific, but I'm currently missing a better term. Maybe `destructive`? Note that it is also possible to add named constant values for true and false that make reading the function call easier. 
```
// in helm.go
const AllowDestructive = true

// in caller.go
helmclient.Upgrade(ctx, config, timeout, helm.AllowDestructive)
// is much clearer to the reader than
helmclient.Upgrade(ctx, config, timeout, true)
```

Reply:
Nice. `AllowDestructive` seems pretty clear to me.